roles/base/tasks/ansible-pull.yml

```diff
@@ -40,6 +40,8 @@
 # ansible-pull expects the inventory file to exist already, even before it
 # clones the repo, so we clone the repo explicitly.
 - name: clone ansible git repository
+  # not that this fails with ansible 2.1.1.0 (working with 2.1.0.0)
```

Comment:
permission denied. enable to read /tmp/ansible<blah>

because the file is written as ssh user (vagrant), and then sudo to bbuser for executing it.
with 2.1.0.0, the file is made world readable after upload.